libs/utils/android/benchmark.py

```diff
@@ -116,7 +116,7 @@ def _parseCommandLine(self):
 
         # Test customization
         parser.add_argument('--results-dir', type=str,
-                default=None,
+                default=self.__class__.__name__,
```

Comment:
> Correct me if I'm wrong but the existing benchmarks don't use DATE_TIME anyway.  Considering that there is some debate here, could we make it a separate issue for a follow-up cleanup?

Sure.

> About the consistency argument, if it has to be made consistent, shouldn't we do in in libs/utils/env.py for all result directories? It appears from that code that its done only if results directory isn't specified which is kind of making it consistent anyway (since I'm a LISA n00b, I could be completely off the mark here)

Yes, I think you're right.